packages/web-app-epub-reader/src/App.vue

```diff
@@ -4,36 +4,76 @@
       <li
         v-for="chapter in chapters"
         :key="chapter.id"
-        class="epub-reader-chapters-list-item oc-mt-xs"
+        class="epub-reader-chapters-list-item oc-py-s"
         :class="{ active: currentChapter.id === chapter.id }"
       >
         <oc-button class="oc-text-truncate" appearance="raw" @click="showChapter(chapter)">
           <span
             v-oc-tooltip="chapter.label"
-            class="oc-text-truncate oc-text-small oc-mr-s"
+            class="oc-text-truncate oc-mr-s"
             v-text="chapter.label"
           />
         </oc-button>
       </li>
     </oc-list>
-    <div class="oc-flex oc-flex-center oc-width-1-1">
-      <div class="oc-flex oc-flex-middle oc-mx-l">
-        <oc-button :disabled="navigateLeftDisabled" appearance="raw" @click="navigateLeft">
-          <oc-icon name="arrow-left-s" fill-type="line" size="xlarge" />
-        </oc-button>
+    <div class="oc-width-1-1 oc-height-1-1">
+      <div class="epub-reader-controls oc-flex oc-flex-middle oc-m-s">
+        <div class="oc-flex oc-button-group">
```

Comment:
Button group breaks into two lines. Also, is it intended that the dropdown spans the full width?

<img width="1151" alt="Screenshot 2024-02-20 at 06 29 59" src="https://github.com/owncloud/web/assets/3532843/7e445503-da76-4013-a2e6-5410249d7304">

Reply:
I think a big select helps on mobile view (iPad) what do you think ? Better on the initial screenshot ?

Reply:
hmm... don't have a hard preference and on even smaller screen it makes sense to use the full width. so maybe best to just keep it like this and wait for feedback.